### DISCUSSION TITLE
[A-L] (12 Giugno 2023) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#391] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile?

$$
    \Large \\{\ \\{p,\ q\\},\ \\{\neg p,\ s\\},\ \\{\neg q,\ s\\}\ \\}.
$$

Se si risponde SI definire un assegnamento che la soddisfa, se si risponde NO dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

**Comment by cek753** [comment]
Sat, q=1 , s=1
